data/mods/Xedra_Evolved/items/tools.json

```
@@ -63,5 +63,124 @@
+  {
+    "id": "dreamforged_halligan",
+    "type": "TOOL",
+    "copy-from": "halligan",
+    "name": { "str": "dreamforged Halligan bar" },
+    "description": "Small tool for breaking into closed locations, forged from dreamdross, which makes it really light.",
+    "proportional": { "weight": 0.25, "melee_damage": { "bash": 0.25 } },
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "forged_dreamstuff" ],
+    "symbol": ";",
+    "color": "light_gray"
+  },
+  {
+    "id": "dreamforged_pliers_locking",
+    "type": "TOOL",
+    "copy-from": "pliers_locking",
+    "name": { "str_sp": "dreamforged locking pliers" },
+    "description": "Strong pilers with adjustable mechanism.  Forged entirely from dreamdross, which makes it much lighter.",
+    "proportional": { "weight": 0.25, "melee_damage": { "bash": 0.25 } },
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "forged_dreamstuff" ],
+    "symbol": ";",
+    "color": "light_gray"
+  },
+  {
+    "id": "dreamforged_pickaxe",
+    "type": "TOOL",
+    "copy-from": "pickaxe",
+    "name": { "str": "dreamforged pickaxe" },
+    "description": "Classic pickaxe, used for digging.  Forged from dreamdross, which makes it really light, albeit a bit weaker.",
+    "proportional": { "weight": 0.3, "melee_damage": { "bash": 0.25 } },
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "forged_dreamstuff" ],
+    "use_action": [ "PICKAXE" ],
+    "symbol": ";",
+    "color": "light_gray"
+  },
+  {
+    "id": "dreamforged_picklocks",
+    "type": "TOOL",
+    "copy-from": "picklocks",
+    "name": { "str": "dreamforged locksmith kit" },
+    "description": "A set of light and sturdy dreamforged lockpicks and wrenches.",
+    "proportional": { "weight": 0.25 },
+    "ascii_picture": "picklocks",
+    "material": [ "forged_dreamstuff" ],
+    "symbol": ";",
+    "color": "light_gray",
+    "qualities": [ [ "LOCKPICK", 10 ] ]
+  },
+  {
+    "id": "dreamforged_scalpel",
+    "type": "TOOL",
+    "copy-from": "scalpel",
+    "name": { "str": "dreamforged scalpel" },
+    "description": "A very sharp medical knife for surgeon operations.  Almost weightless.",
+    "proportional": { "weight": 0.25 },
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "forged_dreamstuff" ],
+    "symbol": ";",
+    "color": "light_gray"
+  },
+  {
+    "id": "dreamforged_e_tool",
+    "type": "TOOL",
+    "copy-from": "e_tool",
+    "name": { "str": "dreamforged entrenching tool" },
+    "description": "Collapsible spade to dig small pits, forged from dreamdross, to make it even lighter than before.",
+    "proportional": { "weight": 0.25 },
+    "price": 0,
+    "price_postapoc": 0,
+    "material": [ "forged_dreamstuff" ],
+    "symbol": ";",
+    "color": "light_gray"
+  },
+  {
+    "id": "dreamforged_ax",
+    "type": "TOOL",
+    "copy-from": "ax",
+    "name": { "str": "dreamforged axe" },
+    "description": "Large, two hand axe to chop some wood.  Light, but still useful",
```

```suggestion
    "description": "Large, two hand axe to chop some wood.  Light, but still useful.",
```
~~Don't axes need weight to chop effectively~~